src/Sulu/Bundle/MediaBundle/Media/FormatManager/FormatManager.php

```diff
@@ -315,6 +315,10 @@ public function getFormatDefinitions($locale = null, array $formatOptions = [])
         $definitionsArray = [];
 
         foreach ($this->formats as $format) {
+            if (array_key_exists('internal', $format) && $format['internal']) {
```

Comment:
For now I simply don't return internal format in this method. Could also be a flag, but flags bloat the signature of methods. I would leave it like this and add another method if we need internal formats also be returned one day.